cli/data.go

```diff
@@ -469,7 +469,7 @@ func (c *viamClient) tabularData(dst string, filter *datapb.Filter) error {
 				},
 				CountOnly: false,
 			})
-			fmt.Fprintf(c.c.App.Writer, ".") // no newline
+			fmt.Fprintf(c.c.App.ErrWriter, ".") // no newline
```

Comment:
Justification: Most cli tools that I know of put progress bars / info on stderr instead of stdout.